src/v2/Apps/Artist/Routes/AuctionResults/ArtistAuctionResults.tsx

```diff
@@ -67,6 +67,20 @@ const AuctionResultsContainer: React.FC<AuctionResultsProps> = ({
   const { hasNextPage, endCursor } = pageInfo ?? {}
   const artistName = artist.name
 
+  const { match } = useRouter()
+  const queryParams: any = paramsToCamelCase(match?.location.query)
```

Comment:
We should try to avoid using `any` wherever we can. Something like this could work:

```
  const { scrollToAuctionResults } = paramsToCamelCase(
    match?.location.query
  ) as { scrollToAuctionResults?: boolean }
  ```

Reply:
Fixed!